.github/workflows/build-sphinx.yml

```diff
@@ -1,26 +1,13 @@
 name: User's Guide
 on:
-    schedule:
-        # Rebuild daily, just after (30 minutes after) any tags were updated.
-        # I'd prefer to rebuild in response to added tags, but that's
-        # much harder to make happen on github actions.
-        - cron: '30 0 * * *'
-
     push:  # Build the docs for any branch (we only push to GCP when building from main)
-        branches:
-            - '*'
-
-    create:  # Any time a tag is added, rebuild the docs.
-        branches:
-            - main
-
     pull_request:  # Build the docs for any PR
-        branches:
-            - '*'
+    create:  # Any time a tag (or branch) is added, rebuild the docs.
+
 jobs:
     build-ug:
         name: Sphinx docs
-        runs-on: macos-10.15
+        runs-on: ubuntu-latest

```

Comment:
macos runners are the least available, using ubuntu where possible might slightly reduce our wait times.